README.md

```diff
@@ -135,48 +173,65 @@ print("Probability with `predict_proba`: ", y_proba_fhe)
+Concrete ML built-in models have APIs that are almost identical to their scikit-learn counterparts. It is also possible to convert PyTorch networks to FHE with the Concrete ML conversion APIs. Please refer to the [linear models](docs/built-in-models/linear.md), [tree-based models](docs/built-in-models/tree.md) and [neural networks](docs/built-in-models/neural-networks.md) documentation for more examples, showing the scikit-learn-like API of the built-in models.
 
-## Online demos and tutorials.
-
-Various tutorials are given for [built-in models](docs/built-in-models/ml_examples.md) and for [deep learning](docs/deep-learning/examples.md) In addition, several complete use-cases are explored:
+<p align="right">
+  <a href="#table-of-contents" > ↑ Back to top </a> 
+</p>
 
-- [Encrypted Large Language Model](use_case_examples/llm/): convert a user-defined part of a Large Language Model for encrypted text generation. Shows the trade-off between quantization and accuracy for text generation and shows how to run the model in FHE.
+## Resources 
 
-- [Credit Scoring](use_case_examples/credit_scoring/): predict the chance of a given loan applicant defaulting on loan repayment while keeping the user's data private. Shows how Concrete ML models easily replace their scikit-learn equivalents
+### 🟨 Demos 
 
-- [Health diagnosis](use_case_examples/disease_prediction/): based on a patient's symptoms, history and other health factors, give
-  a diagnosis using FHE to preserve the privacy of the patient.
+#### Live demos on Hugging Face
 
-- [Private inference for federated learned models](use_case_examples/federated_learning/): private training of a Logistic Regression model and then import the model into Concrete ML and perform encrypted prediction
+- [Credit Scoring](https://huggingface.co/spaces/zama-fhe/credit_card_approval_prediction):  predicting the chance of a given loan applicant defaulting on loan repayment while keeping the user's data private. This demo shows how Concrete ML models easily replace their scikit-learn equivalents.
+  - [code](use_case_examples/credit_scoring/)
+- [Sentiment analysis with transformers](https://huggingface.co/blog/sentiment-analysis-fhe) : predicting if an encrypted tweet / short message is positive, negative or neutral, using FHE.
+  - [code](use_case_examples/sentiment_analysis_with_transformer) and the [blog post](https://huggingface.co/blog/sentiment-analysis-fhe)
+- [Health diagnosis](https://huggingface.co/spaces/zama-fhe/encrypted_health_prediction) : giving a diagnosis using FHE to preserve the privacy of the patient based on a patient's symptoms, history and other health factors.
+  - [code](use_case_examples/disease_prediction/)
+- [Encrypted image filtering](https://huggingface.co/spaces/zama-fhe/encrypted_image_filtering) : filtering encrypted images by applying filters such as black-and-white, ridge detection, or your own filter.
+  - [code](use_case_examples/image_filtering)
```

Comment:
although this is not wrong, use_case_examples/image_filtering actually does not provide any code, they are just links to the HF space

similarly to above, it would be more accurate to suggest https://huggingface.co/spaces/zama-fhe/encrypted_image_filtering/tree/main

but if the goal is to target the Concrete ML github's repository, than ok we can keep this !